go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	k8s.io/apimachinery v0.17.2
 	k8s.io/client-go v0.17.2
 	k8s.io/cri-api v0.0.0
+	k8s.io/klog/v2 v2.0.0
```

Comment:
not a big deal but the latest tag is `v2.4.0`.

Reply:
Ah, ok. Good point. I'll update and check if everything works.